lib/common/memory/src/mmap_ops.rs

```diff
@@ -107,14 +120,15 @@ where
 
     let instant = time::Instant::now();
 
-    let mut dst = [0; 8096];
-
-    for chunk in mmap.chunks(dst.len()) {
-        dst[..chunk.len()].copy_from_slice(chunk);
+    if let Err(err) = mmap.populate() {
+        log::warn!(
+            "Failed to populate mmap{separator}{path:?} to cache: {err}",
+            separator = separator,
+            path = path,
+            err = err
+        );
```

Comment:
Style nit:

```suggestion
        log::warn!(
            "Failed to populate mmap{separator}{path:?} to cache: {err}"
        );
```

---

Also, It was already before this PR, but I don't think this `separator` logic works as expected. If `path` is None, it will print this: `Failed to populate mmap"" to cache: …`. Maybe it should be like this:

```rust
let path = path.map_or_else(|| "".to_string(), |path| format!(" {path:?}"));
…
log::warn!("Failed to populate mmap{path} to cache: {err}");
```